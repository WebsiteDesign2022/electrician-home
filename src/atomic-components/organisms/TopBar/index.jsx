import React from 'react';
import { Navbar, Container } from 'react-bootstrap';
import BarBrand from '../../molecules/BarBrand';
import BarButtons from '../../molecules/BarButtons';

const TopBar = () => (
  <Navbar bg="light" variant="light">
    <Container className=".navbar-expand-lm">

      <BarBrand brandName={"Electrician Home"}/>
      
      <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span className="navbar-toggler-icon"></span>
      </button>

      <div className="collapse navbar-collapse" id="navbarNav">
        <BarButtons />
      </div>

    </Container>
  </Navbar>
);

export default TopBar;
