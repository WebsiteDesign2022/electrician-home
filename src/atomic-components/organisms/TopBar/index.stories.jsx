import React from 'react';

import TopBar from '.';

export default {
  title: 'Electrician-Home/organisms/TopBar',
  component: TopBar,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <TopBar {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {
  
};
