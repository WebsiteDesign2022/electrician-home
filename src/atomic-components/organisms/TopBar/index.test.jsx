import TestRenderer from 'react-test-renderer';

import TopBar from '.';

describe("TopBar", () => {
    test("renders TopBar", () => {
        const topBar = TestRenderer.create(
            <TopBar/>
        );

        expect(topBar.toJSON()).toMatchSnapshot();
    });
});
