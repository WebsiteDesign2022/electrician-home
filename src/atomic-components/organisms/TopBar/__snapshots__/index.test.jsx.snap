// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar renders TopBar 1`] = `
<nav
  className="navbar navbar-expand navbar-light bg-light"
>
  <div
    className=".navbar-expand-lm container"
  >
    <span
      className="px-4  navbar-brand"
    >
      Electrician Home
    </span>
    <button
      aria-controls="navbarNav"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarNav"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarNav"
    >
      <div
        className="justify-content-end navbar-expand-md navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rr-ui-event-key="#home"
          disabled={false}
          href="#home"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#landing"
          disabled={false}
          href="#landing"
          onClick={[Function]}
        >
          Landing
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#service"
          disabled={false}
          href="#service"
          onClick={[Function]}
        >
          Service
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#about"
          disabled={false}
          href="#about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#contact"
          disabled={false}
          href="#contact"
          onClick={[Function]}
        >
          Contact
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#services"
          disabled={false}
          href="#services"
          onClick={[Function]}
        >
          Services
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#project"
          disabled={false}
          href="#project"
          onClick={[Function]}
        >
          Project
        </a>
        <a
          className="nav-link"
          data-rr-ui-event-key="#portfolio"
          disabled={false}
          href="#portfolio"
          onClick={[Function]}
        >
          Portfolio
        </a>
      </div>
    </div>
  </div>
</nav>
`;
